web-ui/view/git_forge.ml

```diff
@@ -55,6 +58,8 @@ module type View = sig
     job:Current_rpc.Job.t ->
     status:Current_rpc.Job.status ->
     csrf_token:string ->
+    timestamps:Run_time.timestamps option ->
+    build_created_at:float option ->
```

Comment:
What is the difference between the creation time and the `queued_at`?

Reply:
https://github.com/ocurrent/ocaml-ci/pull/522/files#diff-95ab9cbeb2c74e16d0ab54903bca0b0a20d108d33a2336be6925fc4871dc22a1R75-R77